components/_buttons/buttons.scss

```diff
@@ -1,7 +1,7 @@
 @mixin light-theme-disabled {
   background: $white;
   border: 0.0625rem solid $gray-4;
-  color: $gray-7;
+  color: rgba(255,255,255,.5);
```

Comment:
Check out _variables.scss, we have variables for these rgba colours 

$rgba-7: rgba(255, 255, 255, 0.5);